src/containers/career.jsx

```diff
@@ -128,13 +130,13 @@ class Career extends Component {
           </div>
         </div>
         <div className="btn">
-          <button id='like' onClick={() => {
+          <img src={dislikeImg} alt="dislike" onClick={() => {
```

Comment:
onClick on a new line 
also im not sure the 'x' image is considered dislike but i guerss it's fine.

Reply:
also here you could get rid of the curly braces cause the arrow function can return without them